### PR TITLE
[IP-696] Filters guest

### PR DESCRIPTION
**Pull request by daniel89fg**
Pull Request Checklist

  * [x] My code follows the code formatting guidelines.
  * [x] I have an issue ID for this pull request. 
  * [x] I selected the corresponding branch.
  * [ ] I have rebased my changes on top of the corresponding branch.
  
Issue Type (Please check one or more)

  * [ ] Bugfix
  * [x] Improvement of an existing Feature 
  * [ ] New Feature

### CODE REVIEW

File: application/modules/filter/controllers/AjaxGuest.php [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+ * @author		InvoicePlane Developers & Contributors
+ * @copyright	Copyright (c) 2012 - 2017 InvoicePlane.com
+ * @license		https://invoiceplane.com/license.txt
+ * @link		https://invoiceplane.com
+ */
+
+/**
+ * Class Ajax
+ */
+class AjaxGuest extends Guest_Controller
```
Comment (Kovah) [inline application/modules/filter/controllers/AjaxGuest.php]: Please standardize class names to `Ajax_Guest` and alter all links to hyphen-separated lowercase links.

File: application/modules/guest/views/index.php [hunk 2]
```diff
@@ -3,32 +3,36 @@
 </div>
 
 <div id="content">
-
+    <?php if (!$this->session->userdata('user_all_clients')) { ?>
```
Comment (Kovah) [inline application/modules/guest/views/index.php]: Is that check correct here? Why did you add it?

File: application/modules/guest/views/invoices_index.php [hunk 3]
```diff
@@ -21,76 +21,12 @@ class="btn  <?php echo $status == 'paid' ? 'btn-primary' : 'btn-default' ?>">
...
-                                </a>
-                            </div>
-                        </td>
-                    </tr>
-                <?php } ?>
-                </tbody>
-
-            </table>
-        </div>
+        <?php $this->layout->load_view('guest/invoices_partial_table', array('invoices' => $invoices)); ?>
```
Comment (Kovah) [inline application/modules/guest/views/invoices_index.php]: Please use the short array syntax using `[` and `]`.

File: application/modules/guest/views/index.php (hunk 2 above)
Comment (daniel89fg) [reply application/modules/guest/views/index.php]: Yeah that's right. it's an improvement that I already published and it's embedded in the system. The users who can see all the clients (who are basically the accountants), do not visualize the budgets since they only need the invoices and the payments.

File: application/modules/guest/views/invoices_index.php (hunk 3 above)
Comment (daniel89fg) [reply application/modules/guest/views/invoices_index.php]: I do not understand

Comment (Kovah) [reply application/modules/guest/views/invoices_index.php]: Instead of `array('invoices' => $invoices)` use the short array syntax `['invoices' => $invoices]` as defined in the [contribution guide](https://github.com/InvoicePlane/InvoicePlane/blob/1.5/CONTRIBUTING.md#php).

Comment (daniel89fg) [reply application/modules/guest/views/invoices_index.php]: ready

File: application/modules/filter/controllers/AjaxGuest.php (hunk 1 above)
Comment (daniel89fg) [reply application/modules/filter/controllers/AjaxGuest.php]: ready

**Comment by UnderDogg** [comment]
Needs label : 1.6 or 1.5.11 or something. Needs Milestone?

**Comment by Kovah** [comment]
It's not scheduled for a version yet.
